tests/framework/checks/skips.go

```diff
@@ -145,18 +145,6 @@ func SkipTestIfNotSEVESCapable() {
 	ginkgo.Skip("no node capable of running SEV-ES workloads detected", 1)
 }
 
-// Deprecated: SkipIfMissingRequiredImage should be converted to check & fail
-func SkipIfMissingRequiredImage(virtClient kubecli.KubevirtClient, imageName string) {
-	windowsPv, err := virtClient.CoreV1().PersistentVolumes().Get(context.Background(), imageName, metav1.GetOptions{})
-	if err != nil || windowsPv.Status.Phase == k8sv1.VolumePending || windowsPv.Status.Phase == k8sv1.VolumeFailed {
-		ginkgo.Skip(fmt.Sprintf("Skip tests that requires PV %s", imageName))
-	} else if windowsPv.Status.Phase == k8sv1.VolumeReleased {
-		windowsPv.Spec.ClaimRef = nil
-		_, err = virtClient.CoreV1().PersistentVolumes().Update(context.Background(), windowsPv, metav1.UpdateOptions{})
```

Comment:
How does this keep working? This skip functions has another responsibility where it recycles the PV.
How does this happen now that it's removed? 😮

Reply:
Isn't it recycled automatically?

Reply:
According to the PV definition, I thought it doesn't. Does the test pass for you?

Reply:
I triggered the lane(it is phase 2). Let's see, I can revert the logic(not the skip) if it is required.

Reply:
Let's see. I think that today we could just use an API on the PersistentVolume that does this https://github.com/kubevirt/kubevirt/pull/12435#discussion_r1705110629

Reply:
Thank you Alex, I reintroduced the commits and appended last commit that switch the skip to fail and I also renamed the function.  We should check for the other option in follow up.

Reply:
Sounds good